ntripclient.c

```diff
@@ -1432,8 +1461,29 @@ int main(int argc, char **argv)
         // handle all other modes
         else
         {
-          if(connect(sockfd, (struct sockaddr *)&their_addr,
-          sizeof(struct sockaddr)) == -1)
+          // Setup a Curl handle to wrap a normal socket to support TLS.
+          // Instead of using "recv/send", use "curl_recv_/curl_send_" instead.
+          curl_global_init(CURL_GLOBAL_DEFAULT);
+          CURL *curl = curl_easy_init();
+          if (curl == NULL)
+          {
+              error = 1;
+              break;
+          }
+
+          // create url string
+          char url[1024];
+          const char *proto = (args.mode == HTTPS) ? "https://" : "http://";
+          snprintf(url, sizeof(url), "%s%s", proto, server);
+          url[sizeof(url)-1] = 0;
+
+          // Set CURL options
+          curl_easy_setopt(curl, CURLOPT_URL, url);
+          curl_easy_setopt(curl, CURLOPT_PORT, atoi(port));
+          curl_easy_setopt(curl, CURLOPT_CONNECT_ONLY, 1L);
+          //curl_easy_setopt(curl, CURLOPT_VERBOSE, 1L);
```

Comment:
Please remove this line if it is no longer used

Reply:
Done. This line is needed time to time to debug the connection during development. I changed the setting to 0. Any developer can change back to 1 for debugging in the future.